src/types/Species.ts

```diff
@@ -6,7 +6,7 @@ export type Species = {
   commonName?: string;
   conservationCategory?: 'CR' | 'DD' | 'EN' | 'EW' | 'EX' | 'LC' | 'NE' | 'NT' | 'VU';
   familyName?: string;
-  growthForm?: GrowthForm;
+  growthForms?: GrowthForm[];
```

Comment:
We can remove line 17 now

Reply:
Sorry I renamed `growthFormNEXT` to `growthForms` to align with the discussion in Slack. I'll go ahead and remove the duplicate `growthForms` property from my mock data PR.

Reply:
You can leave it in there since it is removed by this PR. Yours can get merged in sooner than mine since mine is dependent on some backend stuff.

Reply:
Got it, sounds good 👍